src/js/bootstrap-datetimepicker.js

```diff
@@ -2125,12 +2439,12 @@
             return picker;
         };
 
-        picker.keyBinds = function (keyBinds) {
+        picker.keyBindings = function (keyBinds) {
```

Comment:
I dont think it is a good idea to rename key binds to keyBindings just because it is a breaking change.